src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx

```diff
@@ -54,6 +55,7 @@ export const ArtworkSidebarAuctionPolling: React.FC<Props> = ({
         currentBidChanged={currentBidChanged}
         artwork={artwork}
       />
+      <Spacer my={2} />
```

Comment:
We should be using `mt` or `mb` so that we only get valid spacing scale values. (None of this is obvious; long-standing issue)

Reply:
Also I'd be weary about this: we need to find this exact state. I wouldn't be sure this doesn't compound existing margins which _are already_ correct.

Reply:
@dzucconi I don't quite understand 

> We should be using mt or mb so that we only get valid spacing scale values. (None of this is obvious; long-standing issue)

Can you elaborate? I never use `mt` or `mb` and almost always use `my`. 

Also, the two surrounding fragments _should_ always be there.

Reply:
So like: in the vertical case the margins collapse; which leads to a 'correct' value. But if you inspect you'll see:

![](https://static.damonzucconi.com/_capture/KjGDhOrZ.png)

If you were to use `mx` then you'd just get completely invalid spacing values.

We really need to update the API for this component to prevent invalid values. But yeah generally I just use `mt` and `ml` to avoid this problem.

Reply:
guess `my` on a spacer does top and bottom, which actually doubles the expected padding amount? also, I did the footwork on the state. this is the spot, and this component doesn't return in any other circumstance.

Reply:
Re: the second comment: There's obviously states in which this component looks correct (most of them) and a single state in which it doesn't. Something is wrong. This is perhaps a great example of why you should never ever use external margins in a component.

Reply:
@laurabeth ah OK. Gotcha; thanks!

Reply:
Ah yeah I thought this was the outer component 😅